diff --git a/node_modules/@mightyplow/eleventy-plugin-cache-buster/src/addHashes.js b/node_modules/@mightyplow/eleventy-plugin-cache-buster/src/addHashes.js
index 1612363..636d4bf 100755
--- a/node_modules/@mightyplow/eleventy-plugin-cache-buster/src/addHashes.js
+++ b/node_modules/@mightyplow/eleventy-plugin-cache-buster/src/addHashes.js
@@ -25,7 +25,8 @@ module.exports = function (outputDir, options = defaultOptions) {
     const {
         hashParameter,
         sourceAttributes,
-        createResourceHash
+        createResourceHash,
+        resourceExtensions,
     } = merge(defaultOptions, options);
 
     const resourceTagNames = Object.keys(sourceAttributes);
@@ -41,7 +42,7 @@ module.exports = function (outputDir, options = defaultOptions) {
         const document = parse5.parse(content, parserOptions);
 
         const resourceNodes = findResourceNodes(document)
-            .map((node) => nodeHelper({ sourceAttributes }, node))
+            .map((node) => nodeHelper({ sourceAttributes, resourceExtensions }, node))
             .filter((node) => node.hasValidSource());
 
         const replacements = resourceNodes.map(function (nodeHelper) {
diff --git a/node_modules/@mightyplow/eleventy-plugin-cache-buster/src/createFileChecksum.js b/node_modules/@mightyplow/eleventy-plugin-cache-buster/src/createFileChecksum.js
index 2b02925..9eb06d9 100755
--- a/node_modules/@mightyplow/eleventy-plugin-cache-buster/src/createFileChecksum.js
+++ b/node_modules/@mightyplow/eleventy-plugin-cache-buster/src/createFileChecksum.js
@@ -1,5 +1,6 @@
-const fs = require('fs');
-const md5 = require('md5');
+const fs = require("fs");
+const md5 = require("md5");
+const path = require("path");
 
 /**
  * Reads the content of a file and creates a checksum.
@@ -7,7 +8,11 @@ const md5 = require('md5');
  * @param {string} filePath
  * @return {string}
  */
-module.exports = function createFileChecksum (filePath) {
-    const fileContent = fs.readFileSync(filePath);
-    return md5(fileContent.toString());
+module.exports = function createFileChecksum(filePath) {
+	if (fs.statSync(filePath).isDirectory()) {
+		filePath = path.join(filePath, "index.html");
+	}
+
+	const fileContent = fs.readFileSync(filePath);
+	return md5(fileContent.toString());
 };
diff --git a/node_modules/@mightyplow/eleventy-plugin-cache-buster/src/nodeHelper.js b/node_modules/@mightyplow/eleventy-plugin-cache-buster/src/nodeHelper.js
index df4e938..3a3bf51 100755
--- a/node_modules/@mightyplow/eleventy-plugin-cache-buster/src/nodeHelper.js
+++ b/node_modules/@mightyplow/eleventy-plugin-cache-buster/src/nodeHelper.js
@@ -31,7 +31,9 @@ module.exports = function (options, node) {
         },
 
         hasValidSource () {
-            return node && isRelativeUrl(this.getSource()) ;
+            const { resourceExtensions } = options;
+            const url = this.getSource();
+            return node && isRelativeUrl(url) && resourceExtensions.some(ext => url.endsWith(`.${ext}`));
         },
 
         getStart () {
