diff --git a/node_modules/@mightyplow/eleventy-plugin-cache-buster/src/createFileChecksum.js b/node_modules/@mightyplow/eleventy-plugin-cache-buster/src/createFileChecksum.js
index 2b02925..9eb06d9 100755
--- a/node_modules/@mightyplow/eleventy-plugin-cache-buster/src/createFileChecksum.js
+++ b/node_modules/@mightyplow/eleventy-plugin-cache-buster/src/createFileChecksum.js
@@ -1,5 +1,6 @@
-const fs = require('fs');
-const md5 = require('md5');
+const fs = require("fs");
+const md5 = require("md5");
+const path = require("path");
 
 /**
  * Reads the content of a file and creates a checksum.
@@ -7,7 +8,11 @@ const md5 = require('md5');
  * @param {string} filePath
  * @return {string}
  */
-module.exports = function createFileChecksum (filePath) {
-    const fileContent = fs.readFileSync(filePath);
-    return md5(fileContent.toString());
+module.exports = function createFileChecksum(filePath) {
+	if (fs.statSync(filePath).isDirectory()) {
+		filePath = path.join(filePath, "index.html");
+	}
+
+	const fileContent = fs.readFileSync(filePath);
+	return md5(fileContent.toString());
 };
