diff --git a/node_modules/eleventy-plugin-seo/src/Tags/MetaRobots.js b/node_modules/eleventy-plugin-seo/src/Tags/MetaRobots.js
index bde110f..9262571 100644
--- a/node_modules/eleventy-plugin-seo/src/Tags/MetaRobots.js
+++ b/node_modules/eleventy-plugin-seo/src/Tags/MetaRobots.js
@@ -1,10 +1,12 @@
 const BaseTag = require("./BaseTag");
 
 class MetaRobots extends BaseTag {
-  render(pageNumber) {
+  render(pageNumber, isDraft = false) {
     let robots = "index,follow";
 
-    if (pageNumber > 0) {
+    if (isDraft) {
+      robots = "noindex,nofollow";
+    } else if (pageNumber > 0) {
       robots = `no${robots}`;
     }
 
@@ -19,14 +21,14 @@ class MetaRobots extends BaseTag {
       0
     );
 
-    return Promise.resolve(this.render(pageNumber));
+    return Promise.resolve(this.render(pageNumber, scope.contexts[0].draft));
   }
 
   nunjucksRender(self, context) {
     // Get page number from pagination.
     const pageNumber = self.keyPathVal(context.ctx, "pagination.pageNumber", 0);
 
-    return self.render(pageNumber);
+    return self.render(pageNumber, context.ctx.draft);
   }
 }
 
diff --git a/node_modules/eleventy-plugin-seo/src/Tags/PageDescription.js b/node_modules/eleventy-plugin-seo/src/Tags/PageDescription.js
index 5a000b0..2a89f95 100644
--- a/node_modules/eleventy-plugin-seo/src/Tags/PageDescription.js
+++ b/node_modules/eleventy-plugin-seo/src/Tags/PageDescription.js
@@ -10,16 +10,16 @@ class PageDescription extends BaseTag {
 
   liquidRender(scope, hash) {
     // Get excerpt from front matter.
-    const excerpt = scope.contexts[0].excerpt;
+    const { excerpt, description } = scope.contexts[0];
 
-    return Promise.resolve(this.render(excerpt));
+    return Promise.resolve(this.render(excerpt, description));
   }
 
   nunjucksRender(self, context) {
     // Get excerpt from front matter.
-    const excerpt = context.ctx.excerpt;
+    const { excerpt, description } = context.ctx;
 
-    return self.render(excerpt);
+    return self.render(excerpt, description);
   }
 }
 
