diff --git a/node_modules/eleventy-plugin-seo/src/Description.js b/node_modules/eleventy-plugin-seo/src/Description.js
index 885d262..8d54d8c
--- a/node_modules/eleventy-plugin-seo/src/Description.js
+++ b/node_modules/eleventy-plugin-seo/src/Description.js
@@ -3,10 +3,10 @@ const BaseTag = require("./BaseTag");
 class Description extends BaseTag {
   render(scope, hash) {
     // Get excerpt from front matter.
-    const excerpt = scope.contexts[0].excerpt;
+    const { excerpt, description } = scope.contexts[0];
 
     // Fallback on `description` in config if no excerpt is set for the page.
-    const pageDescription = excerpt || this.siteDescription;
+    const pageDescription = excerpt || description || this.siteDescription;
 
     return Promise.resolve(this.entities.encode(pageDescription));
   }
diff --git a/node_modules/eleventy-plugin-seo/src/MetaRobots.js b/node_modules/eleventy-plugin-seo/src/MetaRobots.js
index e0a486c..a0d9cf7
--- a/node_modules/eleventy-plugin-seo/src/MetaRobots.js
+++ b/node_modules/eleventy-plugin-seo/src/MetaRobots.js
@@ -9,7 +9,12 @@ class MetaRobots extends BaseTag {
       "pagination.pageNumber",
       0
     );
-    if (pageNumber > 0) {
+    
+    const draft = scope.contexts[0].draft || false;
+    
+    if (draft) {
+      robots = "noindex,nofollow";
+    } else if (pageNumber > 0) {
       robots = `no${robots}`;
     }
 
