diff --git a/node_modules/posthtml-img-autosize/lib/imgAutosize.js b/node_modules/posthtml-img-autosize/lib/imgAutosize.js
index 846dff6..d491ba2
--- a/node_modules/posthtml-img-autosize/lib/imgAutosize.js
+++ b/node_modules/posthtml-img-autosize/lib/imgAutosize.js
@@ -111,13 +112,14 @@ function getImageDimensions(imgPath) {
         });
     }
 
-    return new Promise((resolve, reject) => {
-        http.get(imgPath, response => {
-            const chunks = [];
+    // return new Promise((resolve, reject) => {
+    //     http.get(imgPath, response => {
+    //         const chunks = [];
 
-            response
-                .on('data', chunk => chunks.push(chunk))
-                .on('end', () => resolve(imageSize(Buffer.concat(chunks))));
-        }).on('error', reject);
-    });
+    //         response
+    //             .on('data', chunk => chunks.push(chunk))
+    //             .on('end', () => resolve(imageSize(Buffer.concat(chunks))));
+    //     }).on('error', reject);
+    // });
+    return Promise.resolve({});
 }
