diff --git a/node_modules/posthtml-img-autosize/lib/imgAutosize.js b/node_modules/posthtml-img-autosize/lib/imgAutosize.js
index 0d6ac52..a331aa0 100644
--- a/node_modules/posthtml-img-autosize/lib/imgAutosize.js
+++ b/node_modules/posthtml-img-autosize/lib/imgAutosize.js
@@ -102,7 +102,7 @@ function getImageDimensions(imgPath) {
 
     if (!img.host) {
         return new Promise((resolve, reject) => {
-            imageSize(imgPath, (err, dimensions) => {
+            imageSize(imgPath.split("?")[0], (err, dimensions) => {
                 if (err) {
                     reject(err);
                 } else {
