diff --git a/node_modules/posthtml-img-autosize/lib/imgAutosize.js b/node_modules/posthtml-img-autosize/lib/imgAutosize.js
index 846dff6..e1508c1
--- a/node_modules/posthtml-img-autosize/lib/imgAutosize.js
+++ b/node_modules/posthtml-img-autosize/lib/imgAutosize.js
@@ -101,7 +101,7 @@ function getImageDimensions(imgPath) {
 
     if (!img.host) {
         return new Promise((resolve, reject) => {
-            imageSize(imgPath, (err, dimensions) => {
+            imageSize(imgPath.split("?")[0], (err, dimensions) => {
                 if (err) {
                     reject(err);
                 } else {
@@ -111,13 +111,14 @@ function getImageDimensions(imgPath) {
         });
     }
 
-    return new Promise((resolve, reject) => {
-        http.get(imgPath, response => {
-            const chunks = [];
+    // return new Promise((resolve, reject) => {
+    //     http.get(imgPath, response => {
+    //         const chunks = [];
 
-            response
-                .on('data', chunk => chunks.push(chunk))
-                .on('end', () => resolve(imageSize(Buffer.concat(chunks))));
-        }).on('error', reject);
-    });
+    //         response
+    //             .on('data', chunk => chunks.push(chunk))
+    //             .on('end', () => resolve(imageSize(Buffer.concat(chunks))));
+    //     }).on('error', reject);
+    // });
+    return Promise.resolve({});
 }
