diff --git a/node_modules/posthtml-img-autosize/lib/imgAutosize.js b/node_modules/posthtml-img-autosize/lib/imgAutosize.js
index 0d6ac52..f3fc616 100644
--- a/node_modules/posthtml-img-autosize/lib/imgAutosize.js
+++ b/node_modules/posthtml-img-autosize/lib/imgAutosize.js
@@ -55,6 +55,8 @@ module.exports = {
                 promises.push(
                     getImageDimensions(imagePath)
                         .then(dimensions => {
+                            if (!dimensions) return;
+
                             dimensions = ratio(
                                 { width: img.attrs.width, height: img.attrs.height },
                                 dimensions
@@ -68,7 +70,7 @@ module.exports = {
                                 img.attrs.height = dimensions.height;
                             }
                         })
-                        .catch(e => Promise.reject(e.message))
+                        .catch(e => console.error('Erro ao dimensionar', { imagePath }, e))
                 );
 
                 return img;
@@ -120,7 +122,13 @@ function getImageDimensions(imgPath) {
 
             response
                 .on('data', chunk => chunks.push(chunk))
-                .on('end', () => resolve(imageSize(Buffer.concat(chunks))));
+                .on('end', () => {
+                    try {
+                        resolve(imageSize(Buffer.concat(chunks)))
+                    } catch (error) {
+                        console.error('Erro ao dimensionar (sem host)', { imgPath }, error)
+                    }
+                });
         }).on('error', reject);
     });
 }
