diff --git a/node_modules/posthtml-webp/lib/index.js b/node_modules/posthtml-webp/lib/index.js
index b764fd2..3781684 100644
--- a/node_modules/posthtml-webp/lib/index.js
+++ b/node_modules/posthtml-webp/lib/index.js
@@ -13,6 +13,10 @@ module.exports = function (options) {
     options.extensionIgnore = []
   }
 
+  if (options.domainIgnore === undefined) {
+    options.domainIgnore = []
+  }
+
   if (options.replaceExtension === undefined) {
     options.replaceExtension = false
   }
@@ -20,11 +24,12 @@ module.exports = function (options) {
   return function posthtmlWebp (tree) {
     tree.match([{ tag: 'img' }, { tag: 'amp-img' }], function (imgNode) {
       if (imgNode.skip) return imgNode
-      var classes = imgNode.attrs && imgNode.attrs.class && (imgNode.attrs.class.split(' ') || [])
+      var classes = (imgNode.attrs && imgNode.attrs.class && imgNode.attrs.class.split(' ')) || []
       var extension = imgNode.attrs.src.split('.').pop()
       var isIgnoredByClass = options.classIgnore.filter(className => classes.includes(className)).length > 0
       var isIgnoredByExtension = options.extensionIgnore.filter(fileExtension => fileExtension === extension).length > 0
-      var isIgnore = isIgnoredByClass || isIgnoredByExtension
+      var isIgnoredByDomain = options.domainIgnore.some(domain => imgNode.attrs.src.includes(domain))
+      var isIgnore = isIgnoredByClass || isIgnoredByExtension || isIgnoredByDomain
       if (isIgnore) return imgNode
       switch (imgNode.tag) {
         case 'amp-img':
